backend/src/main/java/com/redhat/cloud/notifications/routers/internal/models/InternalUserPermissions.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.cloud.notifications.routers.internal.models;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class InternalUserPermissions {
+    private boolean isAdmin;
+    private final Set<String> applicationIds = new HashSet<>();
```

Comment:
If the SQL query is ordered (see previous comment), this should be changed to an `ArrayList`.